internal/transformer/transformresult.go

```diff
@@ -150,384 +146,389 @@ func transformReadBase(value interface{}, base string, lc logger.LoggingClient)
 	case float64:
 		value = valueFloat64
 	}
-	return value, err
+	return value, nil
 }
 
-func transformReadScale(value interface{}, scale string, lc logger.LoggingClient) (interface{}, error) {
+func transformReadScale(value interface{}, scale string) (interface{}, errors.EdgeX) {
 	switch v := value.(type) {
```

Comment:
The code in this switch looks very similar to previous code in transformparam. Can they be combined?

Reply:
updated to use the same function with additional parameter to determine which direction of transformation to be applied.